pkg/kubelet/network/dns/dns.go

```diff
@@ -280,7 +280,7 @@ func getPodDNSType(pod *v1.Pod) (podDNSType, error) {
 	return podDNSCluster, fmt.Errorf(fmt.Sprintf("invalid DNSPolicy=%v", dnsPolicy))
 }
 
-// Merge DNS options. If duplicated, entries given by PodDNSConfigOption will
+// mergeDNSOptions merge DNS options. If duplicated, entries given by PodDNSConfigOption will
```

Comment:
s/merge/merges/

Reply:
Thank you very much. I have modified it, is that right?

Reply:
👍